frontend/src/static/js/components/test/webstatus-overview-cells.test.ts

```diff
@@ -36,6 +39,19 @@ describe('parseColumnsSpec', () => {
   });
 });
 
+// Add test of parseColumnOptions here
+describe('parseColumnOptions', () => {
+  it('returns default column options when none are specified', () => {
+    const options = parseColumnOptions('');
+    assert.deepEqual(options, DEFAULT_COLUMN_OPTIONS);
+  });
+
+  it('returns an array when given a column options spec', () => {
+    const options = parseColumnOptions('baseline_status_high_date');
+    assert.deepEqual(options, [ColumnOptionKey.BaselineStatusHighDate]);
+  });
+});
+
```

Comment:
Can you add the third test case from this comment
https://github.com/GoogleChrome/webstatus.dev/pull/580/files#r1718780661

Reply:
Parsing the options doesn't change the URL, so there is nothing to test in the context of parseColumnOptions. We have to test where the URL is changed in urls.test.ts.